pkg/object/ks3.go

```diff
@@ -257,9 +257,20 @@ func newKS3(endpoint, accessKey, secretKey, token string) (ObjectStorage, error)
 	}
 	uri, _ := url.ParseRequestURI(endpoint)
 	ssl := strings.ToLower(uri.Scheme) == "https"
-	hostParts := strings.Split(uri.Host, ".")
-	bucket := hostParts[0]
-	region := hostParts[1][3:]
+	var pathStyle bool
+	if uri.Path != "" {
```

Comment:
For sync command, will Path is not empty even with vhost style?

Reply:
yes

Reply:
no，path will empty when use vhost style. The behavior of ks3 of sync command is now consistent with that of s3.